src/main/java/com/example/nasastealphoto/model/Camera.java

```diff
@@ -0,0 +1,51 @@
+package com.example.nasastealphoto.model;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.annotations.Cascade;
+import org.hibernate.annotations.CascadeType;
+import org.hibernate.annotations.CreationTimestamp;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "cameras")
+public class Camera {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
It is better to use **GenerationType.SEQUENCE** strategy with **@SequenceGenerator** annotation and use custom created sequence or default generated one. 

**GenerationType.IDENTITY** specifies that the ID must be set by the DB which will force Hibernate to immediately persist new entity and perform INSERT query, because one of the Hibernat`s rules says that entity should have an ID to be stored in session and managed by Hibernate.

**GenerationType.SEQUENCE** will force Hibernate to select next value of sequence, but not make an insert query

Reply:
Thanks, changed to SEQUENCE